src/styles/modules/_dashboard.scss

```diff
@@ -72,6 +72,13 @@
     padding: 0 1rem 1rem;
   }
 
+  .chart .nodata{
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Opening curly brace `{` should be preceded by one space